keps/prod-readiness/sig-node/3008.yaml

```diff
@@ -0,0 +1,6 @@
+# The KEP must have an approver from the
+# "prod-readiness-approvers" group
+# of http://git.k8s.io/enhancements/OWNERS_ALIASES
+kep-number: 3008
+alpha:
+  approver: ""
```

Comment:
I think you need to choose a PRR approver

Reply:
K, added @dchen1107